### PR TITLE
chore: add release notes for PR 9822

### PR DESCRIPTION
**Pull request by salonig23**
## Description

Added release notes for PR [9822](https://github.com/determined-ai/determined/pull/9822)

## Test Plan

No testing required

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/release-notes/updated-user-roles-slot-caps.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**New Features**
+
+-  WebUI: Allow users with roles Viewer, Editor (and variations), Gen AI and Workspace Admin to be
+   able to view resource quotas for each workspace in the WebUI when RBAC is enabled. When RBAC is
+   disabled, all users can view resource quotas.
```
Comment (tara-det-ai) [inline docs/release-notes/updated-user-roles-slot-caps.rst:7]: WebUI: In the enterprise edition of Determined, give users with Viewer, Editor, GenAI, and Workspace Admin permissions the ability to view resource quotas for each workspace in the WebUI when RBAC is enabled. If RBAC is disabled, all users have access to resource quota information.

Comment (salonig23) [reply docs/release-notes/updated-user-roles-slot-caps.rst]: fixed

File: docs/release-notes/updated-user-roles-slot-caps.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**New Features**
+
+-  WebUI: In the enterprise edition of Determined, give users with Viewer, Editor, GenAI, and
```
Comment (tara-det-ai) [inline docs/release-notes/updated-user-roles-slot-caps.rst:5]:
```suggestion
- WebUI: In the enterprise edition of Determined, give Viewer, Editor, GenAI, and
```

File: docs/release-notes/updated-user-roles-slot-caps.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+   Workspace Admin permissions the ability to view resource quotas for each workspace in the WebUI
```
Comment (tara-det-ai) [inline docs/release-notes/updated-user-roles-slot-caps.rst:6]:
```suggestion
Workspace Admin roles the ability to view resource quotas for each workspace in the WebUI
```

File: docs/release-notes/updated-user-roles-slot-caps.rst [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+   when RBAC is enabled. If RBAC is disabled, all users have access to resource quota information.
```
Comment (tara-det-ai) [inline docs/release-notes/updated-user-roles-slot-caps.rst:7]:
```suggestion
when RBAC is enabled. When RBAC is disabled, all users can view resource quotas.
```

**Review by tara-det-ai — APPROVED** [review]
LGTM

File: docs/release-notes/updated-user-roles-slot-caps.rst [hunk 5]
```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**New Features**
+
+-  WebUI: In the enterprise edition of Determined, give Viewer, Editor, GenAI, and Workspace Admin
```
Comment (amandavialva01) [inline docs/release-notes/updated-user-roles-slot-caps.rst:5]: We also gave the role to the `EditorRestricted` and `EditorProjectRestricted` roles, is that worth mentioning?

Comment (salonig23) [reply docs/release-notes/updated-user-roles-slot-caps.rst:5]: sure or we can do (and adjacent roles)

Comment (amandavialva01) [reply docs/release-notes/updated-user-roles-slot-caps.rst:5]: Yeaa that works as well!

**Pull request merged**